nix/strOrPackage.nix

```diff
@@ -1,12 +1,12 @@
-{ lib, pkgs }:
+{ lib, pkgsets }:
 with lib;
 let
   resolveKey = key:
     let
       attrs = builtins.filter builtins.isString (builtins.split "\\." key);
-      op = sum: attr: sum.${attr} or (throw "package \"${key}\" not found");
+      op = sum: attr: sum.${attr} or sum.nixpkgs.${attr} or (throw "package \"${key}\" not found");
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:**

if pkgsets is
```nix
{
  nixpkgs.bar = {};
  foo.nixpkgs.bar = {};
}
```
it would match `foo.nixpkgs.bar` if  expected pkg is `foo.bar` where error may be expected.

Maybe use [lib.attrset.attrbypath](https://teu5us.github.io/nix-lib.html#lib.attrset.attrbypath)